Non-Interactive-Protocol.md

```diff
@@ -11,115 +11,111 @@
 
 ## Abstract
 
-DLC allows two parties to conduct stateful bitcoin contracts by giving each
-of the parties *cross-signed range of CET transactions*. These CETs transactions
-are all spending the same output from a funding transaction and thus only one of
-CET can confirm. A CET is finalized according to the outcome signed by the previously
-selected oracle.
+DLCs allow two parties to conduct stateful bitcoin contracts by giving each
+other a *counter-signed set of CETs*.
+These CETs all spend the same output from a funding transaction and thus only one
+CET can be confirmed.
+A single CET is finalized when an outcome is signed by a previously chosen oracle.
 
-As the contract is established once funding transaction signatures have been exchanged,
-from then a party should assume counterparty non-cooperation and ensure either
-confirmation of the funding or double-spend of its collatral inputs to avoid exploitation.
+The contract is established once funding transaction signatures have been exchanged.
+From that point on each party should assume their counter-party is non-cooperative and should ensure either
+confirmation of the funding transaction or else double-spend its collateral inputs to avoid exploitation.
 
-Once funding transaction signatures are exchanged, DLC parties aren't required
-to interact anymore. They are three scenarios to consider:
+Once funding transaction signatures are exchanged, DLC parties aren't required to interact anymore.
+There are three scenarios to consider:
 
-1. The funding transaction doesn't confirm, a party must either bump its feerate
-to confirm before DLC maturing or double-spend its funding collateral, thus cancelling the
-DLC.
+1. The funding transaction doesn't confirm, and so a party must either bump its fee-rate
+to confirm before DLC maturation or else double-spend its funding collateral canceling the DLC.
 
-2.  The funding transaction confirms, the oracle releases the outcome signature, a
-participant must broadcast the DLC transaction paired with the outcome.
+2.  The funding transaction confirms, and the oracle releases the outcome signature, so a
+participant must broadcast the CET corresponding to the oracle outcome.
 
-3. The funding transaction confirms, oracle doesn't release the outcome signature, independently
-of counterparty behavior, the party must broadcast the refund transaction after its timelock
-expiration.
+3. The funding transaction confirms, and the oracle doesn't release an outcome signature,
```

Comment:
I would add "before the time lock of the refund transaction expires" (and remove the time lock mention after)

Reply:
but this is talking about after the timelock not before?

Reply:
Yes so my suggestion was:

The funding transaction confirms, and the oracle doesn't release an outcome signature before the time lock of the refund transaction expires, so a party must broadcast the refund transaction.

It sounds clearer to me but feel free to ignore if it does not to you.